### PR TITLE
Add Aniruddha Tapas (fixed grad quote)

### PR DESCRIPTION
**Pull request by Aniruddha-Tapas**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

**Comment by krabhisharma** [comment]
Suggestion :-Check the checklist..

File: _data/AniruddhaTapas/AniruddhaTapas.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Aniruddha Tapas
+institution: Northeastern University
+profile_pic: aniruddha.png
```
Comment (mitulsavani) [inline _data/AniruddhaTapas/AniruddhaTapas.md:4]: image path name seems incorrect, please change it from `aniruddha.png` to `aniruddha.jpg`

File: _data/AniruddhaTapas/AniruddhaTapas.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+quote: Life may be tough at times, but destiny isn't created by the shoes we wear but by the steps we take.
```
Comment (mitulsavani) [inline _data/AniruddhaTapas/AniruddhaTapas.md:5]: the quote has an exact 100 char, great job!

File: _data/AniruddhaTapas/AniruddhaTapas.md (hunk 1 above)
Comment (Aniruddha-Tapas) [reply _data/AniruddhaTapas/AniruddhaTapas.md]: Thanks for letting me know! I have fixed this in my recent commit. Please take a look.

File: _data/AniruddhaTapas/AniruddhaTapas.md (hunk 2 above)
Comment (Aniruddha-Tapas) [reply _data/AniruddhaTapas/AniruddhaTapas.md:5]: Thanks! 😄

**Pull request merged**